metrics.go

```
@@ -165,26 +166,28 @@ func reportTiming(ctx context.Context, fn string, tags ...Tags) (context.Context
+			defer timeout.Stop()
+
+			select {
+			case <-doneC:
+				return
+			case <-timeout.C:
+				clientMux.RLock()
+				defer clientMux.RUnlock()
+
+				err := fmt.Errorf("detected stuck function: %s stuck for %v", name, time.Since(start))
+				fmt.Println(err)
+				client.Incr("func.stuck", tagArray, 1)
```

golangci-lint (2.12.2)

[error] Error return value of `client.Incr` is not checked

(errcheck)